tests/__snapshots__/SshTest__it_can_enable_strict_host_checking__1.txt

```diff
@@ -1,3 +1,3 @@
-ssh  user@example.com 'bash -se' << \EOF-SPATIE-SSH
```

Comment:
This seems in correct, the `-p` flag should not appear if there is no specific port number.

Reply:
Right, missed that, fixed with the next commit.